lib/chart/svg.ex

```diff
@@ -85,7 +85,11 @@ defmodule Contex.SVG do
 
   defp path([], _), do: ""
 
-  defp path(points, false) do
+  defp path(points, nil), do: path(points, :smooth)
+  defp path(points, true), do: path(points, IO.inspect(:smooth))
```

Comment:
Would be good to remove the calls to `IO.inspect()` once we're happy with the changes.

Reply:
Removed the whole distinction on the function level and moved the mapping to the calling function, as you suggested above.